web/components/sections/ReadyToInvest/ReadyToInvest.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-debugger, no-console */
+import React from 'react'
+import PropTypes from 'prop-types'
+import { createTheme, ThemeProvider } from '@mui/material/styles'
+import { Grid, Container, Box, CssBaseline } from '@mui/material'
+import SimpleBlockContent from '../../SimpleBlockContent'
+import styles from './ReadyToInvest.module.css'
+import RedirectButton from '../../RedirectButton/RedirectButton'
+
+
+const theme = createTheme()
+
+function ReadyToInvest(props) {
+  const { heading, buttonText, backgroundColor } = props
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Box sx={{
+        backgroundColor: backgroundColor && backgroundColor
```

Comment:
Needs to set the default color in case the user doesn't fill the background field on the CMS